pkg/render/policyrecommendation.go

```diff
@@ -180,6 +201,26 @@ func (pr *policyRecommendationComponent) deployment() *appsv1.Deployment {
 			Name:  "MULTI_CLUSTER_FORWARDING_CA",
 			Value: pr.cfg.TrustedBundle.MountPath(),
 		},
+		{
+			Name:  "LINSEED_URL",
+			Value: relasticsearch.LinseedEndpoint(pr.SupportedOSType(), pr.cfg.ClusterDomain),
+		},
+		{
+			Name:  "LINSEED_CA",
+			Value: pr.cfg.TrustedBundle.MountPath(),
+		},
+		{
+			Name:  "LINSEED_CLIENT_CERT",
+			Value: pr.cfg.PolicyRecommendationCertSecret.VolumeMountCertificateFilePath(),
+		},
+		{
+			Name:  "LINSEED_CLIENT_KEY",
+			Value: pr.cfg.PolicyRecommendationCertSecret.VolumeMountKeyFilePath(),
+		},
+		{
+			Name:  "LINSEED_TOKEN",
+			Value: GetLinseedTokenPath(pr.cfg.ManagedCluster),
```

Comment:
I assume we have already created the service account for policy rec.

Reply:
Yep, we probably can just set this to the serviceaccount token always, since policy rec doesn't run in managed clusters

Reply:
Yes, the operator creates service account `name: tigera-policy-recommendation, namespace: tigera-policy-recommendation`